doc/source/examples/transformations/center_protein_in_box.ipynb

```diff
@@ -41,7 +41,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        protein_center = protein.center_of_mass(wrap=True)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>thanks, missed that. Fixed.</p>